pyfr/solvers/navstokes/inters.py

```diff
@@ -44,6 +57,41 @@ def __init__(self, *args, **kwargs):
         tplargs = dict(ndims=self.ndims, nvars=self.nvars, rsolver=rsolver,
                        visc_corr=visc_corr, c=self._tpl_c)
 
+        # Generate the additional kernels, view matrices for artificial viscosity
+        self.shock_capturing = self.cfg.get('solver', 'shock-capturing', 'none')
+        if self.shock_capturing == 'artificial-viscosity':
+            be, lhs, rhs = args[0], args[1], args[2]
```

Comment:
Do not use `*args` or `**kwargs`.  This will break if we ever choose to pass some arguments by key.